tests/js/spec/views/settings/projectFiltersAndSampling/transactions.spec.tsx

```diff
@@ -552,16 +482,15 @@ describe('Filters and Sampling - Transaction rule', function () {
 
             // Open Modal
             await renderModal(screen.getByText('Add transaction rule'));
-            const checkedCheckbox = screen.getByRole('checkbox');
 
             // Checked tracing checkbox
-            expect(checkedCheckbox).toBeChecked();
+            expect(screen.getByRole('checkbox')).toBeChecked();
 
             // Uncheck tracing checkbox
-            userEvent.click(checkedCheckbox);
+            userEvent.click(screen.getByRole('checkbox'));
 
             // Unched tracing checkbox
-            expect(checkedCheckbox).not.toBeChecked();
+            expect(screen.getByRole('checkbox')).not.toBeChecked();
```

Comment:
These might get a bit slower as we query the element multiple times. We can play with it in TET-81

Reply:
yes I thought we could tackle it on that other story